docs/api_specs/includes/informational_api_specs.md

```diff
@@ -0,0 +1,389 @@
+# Watcher - Informational API Service
+
+API specification of the Watcher's Informational Service
+
+
+## Account - Get Balance
+
+```shell
+curl http://localhost:4000/account.get_balance -d '{"address": "b3256026863eb6ae5b06fa396ab09069784ea8ea"}'
```

Comment:
Cool. Note that the '-X POST' isn't strictly necessary because curl is smart enough to recognise that it has to be a POST if you're send data. But leaving it in makes it more explicit so I'll put it back everywhere.